blog/2023-12-12-nushell_0_88_0.md

```diff
@@ -80,6 +80,22 @@ Thanks to all the contributors below for helping us making the documentation of
 | ------------------------------------ | ----------- | ------------------------------------------------------- |
 | [@author](https://github.com/author) | ...         | [#12345](https://github.com/nushell/nushell/pull/12345) |
 
+## Spread operator
+
+[#10598](https://github.com/nushell/nushell/issues/10598) asked for a spread operator, and [#11006](https://github.com/nushell/nushell/pull/11006)
+and [#11144](https://github.com/nushell/nushell/pull/11144/) implemented a spread operator in lists and records, respectively.
+The spread operator can help one avoid `append` (when used in lists) and `merge` (when used in records). Spreading arguments to commands has not been implemented yet.
+
+```nushell
+> let x = ["foo"]; [ 1 ...$x ...("foo" | split chars) ...[3 9] [] ] | to nuon
+[1, foo, f, o, o, 3, 9, []]
+> let x = { a: "foo" }; { ...$x y: "bar" ...{ x: 2 } } | to nuon
+{a: foo, y: bar, x: 2}
+```
```

Comment:
i would simply suggest splitting these into two blocks, putting the code on multiple lines, to hopefully make the snippets a bit easier to read :yum:
> suggestion comes after this line

for the example, let's create a list and a record
```nushell
let list = ["foo"]
let record = { a: "foo" }
```
- on lists
```nushell
[
    1,
    ...$x,
    ...("foo" | split chars),
    ...[3 9],
    []
] | to nuon
```
will give
```
[1, foo, f, o, o, 3, 9, []]
```
- on records
```
{
    ...$x,
    y: "bar",
    ...{ x: 2 }
} | to nuon
```
will give
```
{a: foo, y: bar, x: 2}
```

Reply:
Applied, with some minor changes, let me know what you think

Reply:
looks very nice :ok_hand: